openstack/neutron/templates/statefulset-network-agent-apod.yaml

```diff
@@ -1,7 +1,7 @@
 {{- $envAll := . }}
 {{- if .Values.agent.apod | default false }}
 {{- if .Values.global.apods | default false }}
-{{- $release_requires_sec_cap:= or (hasPrefix "ussuri" (default .Values.imageVersion .Values.imageVersionServerAPI)) (hasPrefix "yoga" (default .Values.imageVersion .Values.imageVersionServerAPI)) -}}
+{{- $release_requires_sec_cap:= or (hasPrefix "yoga" (default .Values.imageVersion .Values.imageVersionServerAPI)) (hasPrefix "caracal" (default .Values.imageVersion .Values.imageVersionServerAPI)) -}}
```

Comment:
This looks like a version-switch, but wouldn't this always be true for all our future releases and we could just set this to true?